contracts/test/v0.8/foundry/vrf/VRFV2Plus.t.sol

```diff
@@ -241,6 +249,15 @@ contract VRFV2Plus is BaseTest {
       extraArgs: VRFV2PlusClient._argsToBytes(VRFV2PlusClient.ExtraArgsV1({nativePayment: true}))
     });
     (, uint96 ethBalanceBefore, , ) = s_testCoordinator.getSubscription(subId);
+    vm.expectEmit(true, true, true, true);
+    emit RandomWordsFulfilled(
+      requestId,
+      60034167071501806643072794052071072761405686871992035105939762990515855633675,
+      subId,
+      122504,
+      hex"92fd13380000000000000000000000000000000000000000000000000000000000000001",
```

Comment:
same comment as above. it would be nice to programmatically generate these values so that we don't need to change these whenever we make relevant contract changes

Reply:
Sure

- Used rc.extraArgs here to check against